app/ssttpaffordability/AffordabilityController.scala

```diff
@@ -88,20 +88,32 @@ class AffordabilityController @Inject() (
       val spending = journey.maybeSpending.fold(Seq.empty[Expenses])(_.expenses)
       val income = journey.maybeIncome.fold(Seq.empty[IncomeBudgetLine])(_.budgetLines)
       if (spending.nonEmpty && income.nonEmpty) {
-        Redirect(ssttpaffordability.routes.AffordabilityController.getHowMuchYouCouldAfford())
+        Redirect(ssttpaffordability.routes.AffordabilityController.saveTotalLeftOverIncome())
       } else {
         Future.successful(Ok(views.add_income_spending(income, spending)))
       }
     }
   }
 
+  def saveTotalLeftOverIncome: Action[AnyContent] = as.authorisedSaUser.async { implicit request =>
+    JourneyLogger.info(s"AffordabilityController.getHowMuchYouCouldAfford: $request")
+    journeyService.authorizedForSsttp { journey =>
+      val spending = journey.maybeSpending.fold(Seq.empty[Expenses])(_.expenses)
+      val income = journey.maybeIncome.fold(Seq.empty[IncomeBudgetLine])(_.budgetLines)
+      val totalLeftOverIncome = income.map(_.amount).sum - spending.map(_.amount).sum
+      storeLeftOverIncomeToJourney(totalLeftOverIncome, journey).map { _ =>
+        Redirect(ssttpaffordability.routes.AffordabilityController.getHowMuchYouCouldAfford())
+      }
+    }
+  }
+
   def getHowMuchYouCouldAfford: Action[AnyContent] = as.authorisedSaUser.async { implicit request =>
     JourneyLogger.info(s"AffordabilityController.getHowMuchYouCouldAfford: $request")
     journeyService.authorizedForSsttp { journey =>
       val spending = journey.maybeSpending.fold(Seq.empty[Expenses])(_.expenses)
       val income = journey.maybeIncome.fold(Seq.empty[IncomeBudgetLine])(_.budgetLines)
-      val total = income.map(_.amount).sum - spending.map(_.amount).sum
-      Future.successful(Ok(views.how_much_you_could_afford(income, spending, total)))
+      val totalLeftOverIncome = journey.totalLeftOverIncome
```

Comment:
( in all paces that have `totalLeftOverIncome` )

I first read this as 'left over from what?' Is it from 'remainingIncome'? or `remainingIncomeAfterSpending` (that's way too long :-) )